src/main/java/com/searchcode/app/dao/Repo.java

```diff
@@ -366,7 +373,7 @@ public void addBranchToTable() {
             if(shouldAlter) {
                 List<String> commands = Arrays.asList(
                         "ALTER TABLE \"repo\" RENAME TO \"y6L0VN5j9eQSg65hWtJJ_repo\"",
-                        "CREATE TABLE \"repo\" (\"name\" VARCHAR PRIMARY KEY  NOT NULL ,\"scm\" VARCHAR,\"url\" VARCHAR,\"username\" VARCHAR,\"password\" VARCHAR, \"source\", \"branch\" VARCHAR)",
+                        "CREATE TABLE \"repo\" (\"name\" VARCHAR PRIMARY KEY  NOT NULL ,\"scm\" VARCHAR,\"url\" VARCHAR,\"username\" VARCHAR,\"password\" VARCHAR, \"source\", \"branch\" VARCHAR, \"masks\" VARCHAR)",
```

Comment:
Need to add a check like the above,

             PreparedStatement stmt = conn.prepareStatement("PRAGMA table_info(repo);");
 
             boolean shouldAlter = true;
 
             ResultSet rs = stmt.executeQuery();
             String value = "";
             while (rs.next()) {
                 value = rs.getString("name");
 
                 if("source".equals(value)) {
                     shouldAlter = false;
                 }
             }

Otherwise this code will not run and not add the additional column required.